pom.xml

```diff
@@ -93,6 +93,7 @@
 
   <modules>
     <module>common</module>
+	<module>format</module>
```

Comment:
don't add  <module>format</module> to default module list, it will compile fail if someone don't install thrift.  please only add <module>format</module> under profile "build-with-format". please refer to PR596:https://github.com/apache/incubator-carbondata/pull/596